Neos.Ui/core/src/presentation/Modal.tsx

```diff
@@ -1,47 +1,19 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import cx from 'classnames';
+
+import {Dialog} from '@neos-project/react-ui-components';
 
 export const Modal: React.FC<{
     renderTitle(): React.ReactNode
     renderBody(): React.ReactNode
 }> = props => ReactDOM.createPortal(
-    <div
-        data-ignore_click_outside="true"
-        className={cx(
-            'sg-ig-fixed',
-            'sg-ig-inset-0',
-            'sg-ig-bg-[rgba(0,0,0,.9)]',
-            'sg-ig-animate-overlay-appear',
-            'sg-ig-z-[4]'
-        )}
+    <Dialog
+        isOpen={true}
+        title={props.renderTitle()}
+        onRequestClose={() => {}}
```

Comment:
@grebaldi upsi - should we make the parameter optional? ^^

Reply:
probably :D